### PR TITLE
fix(hub): Fix property accessibility

### PR DESCRIPTION
**Pull request by azabbasi**

### CODE REVIEW

File: sdk/iot/Azure.Iot.Hub.Service/src/Customized/Models/DigitalTwinInterfaces.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+using System.Collections.Generic;
+
+namespace Azure.Iot.Hub.Service.Models
+{
+    /// <summary>
+    /// The DigitalTwinInterfaces.
+    /// </summary>
+    public partial class DigitalTwinInterfaces
+    {
```
Comment (vinagesh) [inline sdk/iot/Azure.Iot.Hub.Service/src/Customized/Models/DigitalTwinInterfaces.cs:12]: Do you need to create a new partial class? We should have PnpInterface defined already. I think you just need to add the property there. The naming was as per what we discussed yesterday.

Comment (azabbasi) [reply sdk/iot/Azure.Iot.Hub.Service/src/Customized/Models/DigitalTwinInterfaces.cs:12]: The property belongs to the DigitalTwinInterfaces class not the PnpInterfface class
PnpInterface is internal so we can't expose a property of that type with public accessibility

Comment (azabbasi) [reply sdk/iot/Azure.Iot.Hub.Service/src/Customized/Models/DigitalTwinInterfaces.cs:12]: am I missing something?

Comment (vinagesh) [reply sdk/iot/Azure.Iot.Hub.Service/src/Customized/Models/DigitalTwinInterfaces.cs:12]: I think we need to delete that class then. I might have made a mistake. There was no class called DigitalTwinInterfaces. There was only Interface.

**Review by vinagesh — APPROVED** [review]

**Pull request merged**